R/retentionIndexing.R

```diff
@@ -52,6 +52,58 @@ indexRtime <- function(x,
   
 }
 
+#' @title 2-point correction of RIs
+#'
+#' @description
+#'
+#' `correctRindex` performs correction of RIs based on reference substances
+#'
```

Comment:
IMHO it would not hurt to expand the description a little bit, also e.g. add an example when this might be needed.